### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by gadjevb**
Task6 is ready for review, you can test the calculator here: https://gadjevb.github.io/

### CODE REVIEW

File: calculator.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+textarea {
```
Comment (freekman) [inline calculator.css]: format

File: calculator.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+var operator = true;
+
+function appendDigit(parameter) {
```
Comment (freekman) [inline calculator.js]: format

File: calculator.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	if(operator){
+		var val = document.getElementById("present").value;
+		var answer = eval(val);
+		document.getElementById("present").value = answer;
+	}else{
+		alert("The equation must end with digit, not operator!");
+	}
+}
+
+function remove() {
```
Comment (freekman) [inline calculator.js]: when i click the ```R``` it disappears

Comment (gadjevb) [inline calculator.js]: What disappears?

**Pull request closed without merging**